packages/dashboard/src/helpers/filterData.ts

```diff
@@ -24,8 +24,9 @@ function getMaxTierAndSetFilterTiers(filters: SharedFilter[]): number {
 }
 
 function filter(data = [], filters: SharedFilter[], condition) {
+  const filteredFilters = filters.filter(f => f.resetLabel !== f.active)
```

Comment:
@joshlacey @Atash3000 would activeFilters be a better variable name here?

Reply:
💯